integration-tests/smoke/keeper_test.go

```diff
@@ -425,6 +429,85 @@ func getKeeperSuite(
 					}, "1m", "1s").Should(Succeed())
 				})
 			}
+
+			if testToRun == AddFundsToUpkeepTest {
+				It("registers a new upkeep which is underfunded (and hence doesn't get executed initially), "+
+					"then adds funds and watches it perform", func() {
```

Comment:
let's make the test name shorter like `add funds to new underfunded upkeep`

Reply:
Sure, updated it now :blush: